modules/ROOT/pages/preview-community.adoc

```diff
@@ -0,0 +1,19 @@
+= Preview Your API Community
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+To show a live preview of the community's site, including APIs in *Preview Mode*, open the ACM administrator interface and select *Community Builder*.
```

Comment:
Rewrote to use task steps, and refer to APIs as those visible only to admins, rather than those in "Preview Mode". ("Preview Mode" is a UI setting, but in an entirely different part of the UI, at the menu where admins control which APIs are in preview mode.)